src/f4enix/input/d1suned.py

```
@@ -365,6 +367,147 @@ def remove_irradiation_time(self, index: int) -> None:
+            how="left",
+        )
+
+        # Calculate rescaled dose contribution for each daughter
+        merged["Delta Rescaled Contribution"] = (
+            (merged["TCF_new"] - merged["TCF_ref"]) / merged["TCF_ref"]
+        ) * merged["Normalized Value"]
+
+        merged["Rescaled Normalized Contribution"] = (
+            merged["TCF_new"] * merged["Normalized Value"] / merged["TCF_ref"]
+        )
+        if dtype == float:
```

Ruff (0.14.8)

Use `is` and `is not` for type comparisons, or `isinstance()` for isinstance checks

(E721)